lib/geckoboard/datasets_client.rb

```
@@ -31,5 +31,13 @@ def put_data(dataset_id, data)
     def dataset_path(dataset_id)
       "/datasets/#{CGI.escape(dataset_id)}"
     end
+
+    def hashify_fields(fields)
+      return fields if fields.is_a? Hash
```

Should we raise an exception if `fields` is not a `Hash` or `Array`?